oracle/src/constants.rs

```diff
@@ -0,0 +1,41 @@
+pub trait SpongeConstants {
+    const SPONGE_CAPACITY: usize = 1;
+    const SPONGE_WIDTH: usize = 3;
+    const SPONGE_RATE: usize = 2;
+    const PERM_ROUNDS_FULL: usize;
+    const PERM_ROUNDS_PARTIAL: usize;
+    const PERM_HALF_ROUNDS_FULL: usize;
+    const PERM_SBOX: u32;
+    const PERM_FULL_MDS: bool;
+    const PERM_INITIAL_ARK: bool;
+}
+
+#[derive(Clone)]
+pub struct PlonkSpongeConstantsLegacy {}
```

Comment:
for example, this is what Mina uses for wallets (for signing transactions I believe cc @jspada ). Whereas the other constants are what is used by Mina for building proofs